omnigibson/robots/behavior_robot.py

```diff
@@ -430,17 +432,17 @@ def teleop_data_to_action(self, teleop_action) -> np.ndarray:
         Total size: 44
         """
         # Actions are stored as 1D numpy array
-        action = np.zeros(self.action_dim)
+        action = th.zeros(self.action_dim)
         # Update body action space
         if teleop_action.is_valid["head"]:
             head_pos, head_orn = teleop_action.head[:3], T.euler2quat(teleop_action.head[3:6])
-            des_body_pos = head_pos - np.array([0, 0, m.BODY_HEIGHT_OFFSET])
-            des_body_rpy = np.array([0, 0, R.from_quat(head_orn).as_euler("XYZ")[2]])
+            des_body_pos = head_pos - th.tensor([0, 0, m.BODY_HEIGHT_OFFSET])
+            des_body_rpy = th.tensor([0, 0, T.quat2euler(head_orn.unsqueeze(0))[2][0]])
```

Comment:
no need to unsqueeze because your function already accommodate both? Same applies to many places that use `T.quat2euler`.